심사문제13.7.py

```diff
@@ -0,0 +1,18 @@
+#표준 입력으로 가격(정수)과 쿠폰 이름이 각 줄에 입력됩니다. 
+# Cash3000 쿠폰은 3,000원, Cash5000 쿠폰은 5,000원을 할인합니다. 
+# 쿠폰에 따라 할인된 가격을 출력하는 프로그램을 만드세요(input에서 안내 문자열은 출력하지 않아야 합니다).
+# 7줄 작성
+# 예)
+# 입력) 27000
+# 입력) Cash3000
+# 결과) 24000
+# 입력) 72000
+# 입력) Cash5000
+# 결과) 67000
+a = int(input())
+b = input()
```

Comment:
a와 b가 무엇을 나타내는지 알 수 없습니다!
변수명을 적절히 알맞은 것을 사용해 어떤 의미인지 파악할 수 있게 해주세요!
```suggestion
price = int(input())
coupon = input()
```